### DISCUSSION TITLE
TCP port 2049

### DISCUSSION THREAD

**Discussion by prince-taker** [chpc-tech-eval/scc#142] [Q&A] [answered]
Please assist on how we are supposed to add the tpc port 2049 to the hn.nft. Is my attempt correct?

![image](https://github.com/chpc-tech-eval/chpc24-scc-nmu/assets/69721467/f4cc8e46-5f81-459d-b31f-2f4b6212d592)

**Comment by WilliamMohlahlane** [comment, marked as answer]
### **ThunderBytes**

Have you tried removing the underscore on the tcp port? That could be the problem
